src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -567,23 +565,31 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
                 continue;
             }
 
-            // We get the name and the path to graph as a pair separated by |.
+            // We get the name .
             char topic_name[FRONTEND_DATA_LENGTH];
             bzero(topic_name, FRONTEND_DATA_LENGTH + 1);
-
             read(connFd, topic_name, FRONTEND_DATA_LENGTH);
 
+            string con_message = "successfully get kafka topic";
```

Comment:
Is it possible to have a shorten version of a command here? That will reduce the network cost.

Reply:
Ok sir